### PR TITLE
Job posting redesign part two

### PR DESCRIPTION
**Pull request by omartehsin1**
# Summary | Résumé

Job posting redesign. There was a merge issue with the previous branch that caused some CSS issues, this branch takes care of it for the job postings redesign. 

- Added in the reverted CSS styling
- Added in reverted job posting code
- Added role="presentation" to right arrow images

[Figma](https://www.figma.com/file/LW1JsT0h3IY2trMHdIVDQc/CDS-Website-Revamp-(Work-in-Progress)?node-id=132%3A8131)

### CODE REVIEW

File: assets/sass/cds/_components.scss [hunk 1]
```diff
@@ -154,6 +154,17 @@
   a:link {
     color: #245269;
   }
+	background: #d7faff;
```
Comment (dj2) [inline assets/sass/cds/_components.scss:157]: nit: can you fix the indenting to match?

File: assets/sass/cds/_components.scss [hunk 2]
```diff
@@ -154,6 +154,17 @@
...
+	border-left: 4px solid #269abc;
+	padding: 2.2rem 2.5rem;
```
Comment (dj2) [inline assets/sass/cds/_components.scss:159]: The info-alert also shows up on the spa research page, does that stying still look ok?

File: assets/sass/cds/_components.scss [hunk 3]
```diff
@@ -253,6 +264,98 @@
       background: $primary-color;
     }
   }
+	list-style: none;
```
Comment (dj2) [inline assets/sass/cds/_components.scss:267]: Nit: formatting

Can this bit setting the values for .team-listing be moved up with the others on line 182?

Comment (dj2) [reply assets/sass/cds/_components.scss:267]: Actually, is this all duplicated from above?

File: assets/sass/cds/_components.scss [hunk 4]
```diff
@@ -343,7 +446,7 @@
     font-weight: 100;
     margin-top: 0;
     margin-top: 1.5rem;
-    color: $black;
+    color: $primary-color;
```
Comment (dj2) [inline assets/sass/cds/_components.scss:449]: This should not change, it will break the blog pages.

File: assets/sass/cds/_components.scss [hunk 5]
```diff
@@ -384,7 +487,7 @@
 
     a {
       text-decoration: none;
-      color: $black;
+      color: $white;
```
Comment (dj2) [inline assets/sass/cds/_components.scss:490]: ditto

File: assets/sass/cds/_components.scss [hunk 6]
```diff
@@ -454,9 +557,188 @@
       width: 70%;
     }
   }
+	h1 {
```
Comment (dj2) [inline assets/sass/cds/_components.scss:560]: nit: formatting

File: assets/sass/cds/_components.scss [hunk 7]
```diff
@@ -454,9 +557,188 @@
...
+		font-weight: 100;
+		margin-top: 0;
+		margin-top: 1.5rem;
+		color: $black;
+		margin-left: 0;
+		margin-right: 0;
+		padding-left: 0;
+	}
+
+	h2{
```
Comment (dj2) [inline assets/sass/cds/_components.scss:571]: These change the meet-the-team page, I thought this was for the job posting page?

File: assets/sass/cds/_components.scss [hunk 8]
```diff
@@ -454,9 +557,188 @@
...
   font-size: 45px;
   margin-bottom: 4rem;
 }
+
+.job-postings-container {
+  width: 65%;
+  margin-left: 2.5rem;
+  
+  .heading-title {
+	padding: 0 !important;
```
Comment (dj2) [inline assets/sass/cds/_components.scss:691]: nit: formatting

File: assets/sass/cds/_sections.scss [hunk 9]
```diff
@@ -417,6 +417,7 @@ span.livefr {
     }
   }
 
+  h5,
```
Comment (dj2) [inline assets/sass/cds/_sections.scss:420]: I don't think this is needed is it?

File: assets/sass/cds/_sections.scss [hunk 10]
```diff
@@ -508,6 +509,9 @@ span.livefr {
     background-color: $discovery-color;
   }
 
+  h4.discovery::after {
```
Comment (dj2) [inline assets/sass/cds/_sections.scss:512]: This is duplicated with below.

File: assets/sass/cds/_sections.scss [hunk 11]
```diff
@@ -526,6 +530,24 @@ span.livefr {
     background-color: $beta-color;
   }
 
+  h4.beta::after {
```
Comment (dj2) [inline assets/sass/cds/_sections.scss:533]: Why are these added?

File: assets/sass/cds/_sections.scss [hunk 12]
```diff
@@ -1469,3 +1430,75 @@ article.post {
     margin-bottom: 0;
   }
 }
+.work-with-us--apply-now {
+  width: 65%;
```
Comment (dj2) [inline assets/sass/cds/_sections.scss:1434]: This doesn't seem like just a move, are they changes here expected for the job page?

**Review by dj2 — COMMENTED** [review]
Can you update the title/description of the PR to reflect that this is adding the job page back in?

File: layouts/section/job-posting.en.html [hunk 13]
```diff
@@ -1,145 +1,154 @@
...
-              ><i
-                style="display:inline-block;margin-right:10px;"
-                class="fas fa-spinner fa-spin"
-              ></i></span
-            >Submit
-          </button>
-          <a href="/legal/privacy-notice">Privacy Notice</a>
-        </form>
+        <h2 class="h2-font-weight">Our hiring process</h2>
+        <h1 class="cds-title heading-title"> At CDS, we welcome and invite difference.</h1>
```
Comment (CalvinRodo) [inline layouts/section/job-posting.en.html:44]: I'm curious as to why we have two `h1` tags on this page. 
While it's totally fine to have them from an a11y point of view even if it's discouraged on most GoC sites it can be confusing from a semantic point of view. 


Looking at the rendered page I'm thinking maybe we are missing something as having an `h2` with no content and then an h1 directly after is an a11y issue. Are we missing content or is the second h1 marked up improperly? 


image

File: layouts/section/job-posting.en.html [hunk 14]
```diff
@@ -1,145 +1,154 @@
...
+        <p>CDS welcomes all applicants, including Veterans and people of all races, ethnicities, religions, sexual
+          orientations, gender identities and expressions, national origins, disabilities, ages, body sizes and
+          including those with diverse households and family commitments. We are committed to providing an inclusive and
+          barrier-free work environment, starting with the hiring process. If you need to be accommodated during any
+          phase of the evaluation process, please use the Contact information below to request specialized
+          accommodation. All information received in relation to accommodation will be kept confidential.</p>
+
+        <div class="cta__link black">
+
+          <a href="/join-our-team/">More about our hiring process<img src="/img/icons/arrow-right-white.svg" role="presentation"/></a>
```
Comment (CalvinRodo) [inline layouts/section/job-posting.en.html:62]: please add `alt=""` to the `img` tag so it's properly backwards compatible.

File: layouts/section/job-posting.en.html [hunk 15]
```diff
@@ -1,145 +1,154 @@
...
+
+        </div>
+
+        <textarea class="form-control validate-required validate-maxwords" id="samples" name="samples" rows="5"
+          aria-required="true" aria-describedby="body-required word-count-message"></textarea>
+      </div>
+      <!-- Submit-->
+      <div>
+        <button type="submit" class="submit-btn">
+          Submit<img src="/img/icons/arrow-right-white.svg" role="presentation"/>
```
Comment (CalvinRodo) [inline layouts/section/job-posting.en.html:138]: please add `alt=""` to the `img` tag so it's properly backwards compatible.

File: layouts/section/job-posting.fr.html [hunk 16]
```diff
@@ -1,144 +1,151 @@
...
-              ></i></span
-            >Soumettre
-          </button>
-          <a href="/transparence/enonce-de-confidentialite/"
-            >Énoncé de confidentialité</a
-          >
-        </form>
+        <div class="cta__link black">
+          <a href="/join-our-team/">En savoir plus sur notre processus de recrutement<img
+              src="/img/icons/arrow-right-white.svg" role="presentation"/></a>
```
Comment (CalvinRodo) [inline layouts/section/job-posting.fr.html:63]: please add `alt=""` to the `img` tag so it's properly backwards compatible.

File: layouts/section/job-posting.fr.html [hunk 17]
```diff
@@ -1,144 +1,151 @@
...
+          <span class="url">Liens URL SVP!</span>
+
+        </div>
+        <textarea class="form-control validate-required validate-maxwords" id="samples" name="samples" rows="5"
+          aria-required="true" aria-describedby="body-required word-count-message"></textarea>
+      </div>
+      <!-- Submit-->
+      <div>
+        <button type="submit" class="submit-btn">
+          Soumettre<img src="/img/icons/arrow-right-white.svg" role="presentation"/>
```
Comment (CalvinRodo) [inline layouts/section/job-posting.fr.html:136]: please add `alt=""` to the `img` tag so it's properly backwards compatible.

**Review by CalvinRodo — CHANGES_REQUESTED** [review]
Please address comments, thanks.

**Comment by omartehsin1** [comment]
@dj2, I added in the css files that were reverted. Am I supposed to only add in the styling for the job posting?

File: assets/sass/cds/_sections.scss (hunk 9 above)
Comment (omartehsin1) [reply assets/sass/cds/_sections.scss:420]: This was from the reverted CSS

File: assets/sass/cds/_sections.scss (hunk 10 above)
Comment (omartehsin1) [reply assets/sass/cds/_sections.scss:512]: This was from the reverted CSS

File: assets/sass/cds/_sections.scss (hunk 11 above)
Comment (omartehsin1) [reply assets/sass/cds/_sections.scss:533]: This was from the reverted CSS

File: assets/sass/cds/_sections.scss (hunk 12 above)
Comment (omartehsin1) [reply assets/sass/cds/_sections.scss:1434]: Yes: https://www.figma.com/file/LW1JsT0h3IY2trMHdIVDQc/CDS-Website-Revamp-(Work-in-Progress)?node-id=132%3A8131

File: layouts/section/job-posting.en.html (hunk 13 above)
Comment (omartehsin1) [reply layouts/section/job-posting.en.html:44]: Can we have multiple <h2> tags?

File: assets/sass/cds/_sections.scss [hunk 18]
```diff
@@ -1469,3 +1407,76 @@ article.post {
     margin-bottom: 0;
   }
 }
+
+.work-with-us--apply-now {
```
Comment (dj2) [inline assets/sass/cds/_sections.scss:1411]: Can you move this block back up to where it was to make it easier to see the difference?

File: layouts/section/job-posting.fr.html [hunk 19]
```diff
@@ -1,144 +1,151 @@
...
-  <section class="content">
-    <section class="section page--outer-container-padding">
+<div class="job-postings-container">
+  <section>
+    <section class="padding">
       <div class="row">
         <div class="col-sm-10 col-sm-offset-1 col-xs-12">
-          <h2 class="section--title">{{ i18n "hiring" }}</h2>
-          {{ .Content }}
+          {{ if .Params.leverId }}
```
Comment (dj2) [inline layouts/section/job-posting.fr.html:7]: If there is no lever ID we're going to have a post that skips the `h1` and goes straight to `h2`.

Comment (dj2) [reply layouts/section/job-posting.fr.html:7]: This if seems to be in a different position in the english version?

File: assets/sass/cds/_sections.scss (hunk 18 above)
Comment (omartehsin1) [reply assets/sass/cds/_sections.scss]: Done

File: layouts/section/job-posting.fr.html (hunk 19 above)
Comment (omartehsin1) [reply layouts/section/job-posting.fr.html:8]: Sorry! forgot to change the fr version of it, en version is correct. Now for both EN and FR, the "apply now" button will only exist if there is a leverId

**Review by CalvinRodo — APPROVED** [review]
It looks good to me

**Review by CalvinRodo — COMMENTED** [review]
Just the double label issue.
